assembl/static2/js/app/utils/inputHandler.js

```diff
@@ -0,0 +1,12 @@
+/*
+  Utility method to handle input events fed from a component via a onChange.
+  @params component [React] The component in question
+  @params inputEvent [SyntheticEvent] The event that is triggered from the onChange
+  @returns void
+*/
+export default (component, inputEvent) => {
```

Comment:
Yes, you add an onChange handler for each attribute. Then you have to create many methods to manage the onChange, or create one, and then you have duplication of code. So, I abstracted the duplicated piece of code and put it here. It's to simplify the process, and it is not a globally unique thing. It's only used for form inputs. I'll add comments to be more clear.